pkg/cmd/attestation/verify/verify.go

```diff
@@ -23,8 +23,10 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 	verifyCmd := &cobra.Command{
 		Use:   "verify [<file-path> | oci://<image-uri>] [--owner | --repo]",
 		Args:  cmdutil.ExactArgs(1, "must specify file path or container image URI, as well as one of --owner or --repo"),
-		Short: "Verify an artifact's integrity using attestations",
+		Short: "(BETA) Verify an artifact's integrity using attestations",
 		Long: heredoc.Docf(`
+			# BETA: Feature subject to change
```

Comment:
I don't totally understand how `cobra` works. This will show up if I do `gh attestation verify --help` but not `gh attestation verify`?

Reply:
Yeah, the `Long` field shows up when `--help` is used for a command OR `gh reference` which is a custom command that lists all GH CLI commands help in 1 place

Reply:
As an aside, recommend checking out https://cobra.dev/

Reply:
And also visible in the online manual e.g. https://cli.github.com/manual/gh_auth_login

Reply:
Also worth calling out that `Short` shows up when calling `gh at --help`:

```shell
$ ./bin/gh at --help
# BETA: Feature subject to change

Download and verify artifact attestations.


USAGE
  gh attestation [subcommand] [flags]

ALIASES
  at

AVAILABLE COMMANDS
  download:    (BETA) Download an artifact's Sigstore bundle(s) for offline use
  verify:      (BETA) Verify an artifact's integrity using attestations

INHERITED FLAGS
  --help   Show help for command

LEARN MORE
  Use `gh <command> <subcommand> --help` for more information about a command.
  Read the manual at https://cli.github.com/manual
```